protostar/commands/test/test_environment_exceptions.py

```diff
@@ -158,7 +158,11 @@ def match(self, other: "RevertableException") -> bool:
     def can_pattern_be_found(pattern: str, strings: List[str]) -> bool:
         return any(pattern in string for string in strings)
 
-    def __reduce__(self):
+    def __reduce__(
+        self,
+    ) -> Tuple[
+        Type, Tuple[Union[List[str], Optional[str], Optional[int]], ...], Dict[str, Any]
+    ]:
```

Comment:
```suggestion
    def __reduce__(self) -> Any:
```

I would avoid typing this monster